core/src/main/java/org/apache/carbondata/core/datastore/impl/FileFactory.java

```diff
@@ -43,7 +44,7 @@
    * LOGGER
    */
   private static final Logger LOGGER =
-      LogServiceFactory.getLogService(FileFactory.class.getName());
+          LogServiceFactory.getLogService(FileFactory.class.getName());
```

Comment:
There are lots of unnecessary format changes, please remove those changes and keep only the changes required for this PR